Makefile

```diff
@@ -1414,6 +1414,8 @@ LINT_CPP_EXCLUDE += $(LINT_CPP_ADDON_DOC_FILES)
 # These files were copied more or less verbatim from V8.
 LINT_CPP_EXCLUDE += src/tracing/trace_event.h src/tracing/trace_event_common.h
 
+# deps/ncrypto is included in this list, as it lives in
+# this repository, and therefore is linted.
```

Comment:
**Review note:** AFAIK the comment can't go within the `wildcard` statement, so that's why it's up here.

Reply:
```suggestion
# deps/ncrypto is included in this list, as it lives in
# this repository, and therefore is linted.
LINT_CPP_DEPS = deps/ncrypto/*.cc deps/ncrypto/*.h
```